app/javascript/Greetings.js

```diff
@@ -0,0 +1,26 @@
+import React, { useState, useEffect } from "react";
+
+const Greetings = () => {
+  const [greetings, setGreetings] = useState([]);
+  const [randomGreeting, setRandomGreeting] = useState("");
+
+  useEffect(() => {
+    fetch("/greetings.json")
+      .then((response) => response.json())
+      .then((data) => {
+        setGreetings(data);
+        const randomIndex = Math.floor(Math.random() * data.length);
+        setRandomGreeting(data[randomIndex]?.body || "");
```

Comment:
- [ ] As mentioned by the previous :point_up: reviewer please remove this line of code because you are already generating random greetings in your backend so you don't need that code here also remove line no. 5 as well :slightly_smiling_face: